datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -74,17 +88,18 @@ $ docker run --rm -i --name ucp \
 $ gpg --decrypt /tmp/backup.tar | tar --list
 ```
 
-## Restore your cluster
+## Restore the swarm
 
-The restore command can be used to create a new UCP cluster from a backup file.
-When restoring, make sure you use the same version of the `docker/ucp` image that you've used to create the backup.
-After the restore operation is complete, the following data will be recovered
-from the backup file:
+The first step in restoring UCP from a backup, is to
```

Comment:
You don't need the comma. Also, why not just number the steps, then you can avoid saying "The first step in restoring...." at all.